forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -84,6 +88,7 @@ internal class BTPinCollector(
             path = capturePaymentPath(paymentRef)
         }
 
+        logger.i("Sending payment capture to BasisTheory")
```

Comment:
include `merchantAccount`,  `paymentRef` and `cardToken` in log?

Reply:
https://github.com/teamforage/forage-android-sdk/pull/42#discussion_r1274193522

should omit cardToken